master/master.cfg

```diff
@@ -287,6 +294,13 @@ c['schedulers'] = [
             branch_re=(gerrit_branch + "/*"),
             eventtype="patchset-created"),
         builderNames=[builder.name for builder in tarball_builders]),
+    # schedule new tags
+    schedulers.AnyBranchScheduler(
+        name="tag-changes",
+        change_filter=util.ChangeFilter(
+            project=gerrit_project,
+            branch_re="^refs/tags/[0-9]+\.[0-9]+\.[0-9]+$"),
```

Comment:
This regex is too restrictive.  See Documentation/versioning.txt on the Lustre master branch.